src/base/definitions.jl

```diff
@@ -40,32 +40,32 @@ Inverter Inner Vars:
 md_var :: Modulation signal on the d-component
 mq_var :: Modulation signal on the q-component
 Vdc_var :: DC voltage supplied by the DC source
-Vd_filter_var :: Voltage seen in the capacitor of the filter in the d-component
-Vq_filter_var :: Voltage seen in the capacitor of the filter in the q-component
+Vr_filter_var :: Voltage seen in the capacitor of the filter in the R-component
+Vi_filter_var :: Voltage seen in the capacitor of the filter in the I-component
 ω_freq_estimator_var :: Frequency estimated by the frequency estimator.
 V_oc_var :: Control voltage supplied from the outer loop control to the inner loop
 ω_oc_var :: Control frequency supplied from the outer loop control the inner loop
 θ_oc_var :: Variation of the angle (PLL or VSM) of the inverter
 VR_inv_var :: Real terminal voltage on the inverter
 VI_inv_var :: Imaginary terminal voltage on the inverter
-Vd_cnv_var :: Voltage supplied from the converter in the d-component
-Vq_cnv_var :: Voltage supplied from the converter in the q-component
+Vr_cnv_var :: Voltage supplied from the converter in the R-component
+Vi_cnv_var :: Voltage supplied from the converter in the I-component
 """
 
 @enum inverter_inner_vars begin
     md_var = 1
     mq_var = 2
     Vdc_var = 3
-    Vd_filter_var = 4
-    Vq_filter_var = 5
+    Vr_filter_var = 4
+    Vi_filter_var = 5
     ω_freq_estimator_var = 6
     V_oc_var = 7
```

Comment:
Do we still need these?

Reply:
Probably. That formulation allow us to do choose to do the power tracking on the filter or in the bus voltage. I'm thinking in maybe in changing V_filter to V_tracking or similar, that determinate where are we measuring. In the case of bus voltage V_tracking = V_bus, and in the case of D'Arco filter we have V_tracking = V_filter. But, we can discuss that change on the other PR.

Also I wouldn't eliminate those just in case we want to implement an algebraic filter. I think it makes sense to use an algebraic filter if we are modeling all lines as algebraic.